lib/101/stack_learn.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class StackLearn extends StatelessWidget {
+  const StackLearn({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          Container(
+            color: Colors.red,
+            height: 100,
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: 50),
+            child: Container(
+              color: Colors.blue,
+              height: 100,
+            ),
+          ),
+          Positioned(
+            bottom: 0,
+            width: 25,
+            height: 100,
+            child: Container(
+              color: Colors.green,
+            ),
+          ),
+          Positioned(
+            bottom: 0,
+            width: 25,
+            height: 100,
+            child: Container(
+              color: Colors.green,
+            ),
+          ),
```

Comment:
Why these two child widgets are in the same position and are exactly the same, is there any reason? If not, please remove the last one.

Reply:
It should be Positioned.fill will fix the code and update.